.github/workflows/dhis2-verify-app.yml

```diff
@@ -175,16 +177,36 @@ jobs:
 
     release:
         runs-on: ubuntu-latest
-        needs: [publish]
-        if: "github.event_name == 'push' && !contains(github.event.head_commit.message, '[skip ci]')"
+        needs: [build, lint, test] # add e2e if you use it
```

Comment:
you can also remove the `publish` action above, since the new `release` one also does publishing